warehouse-service/src/main/java/com/codecool/warehouse/api/CheckoutApiClient.java

```diff
@@ -36,4 +36,19 @@ public CachedProduct getReservedAmountFromOrderService(UUID itemPID) {
             );
         }
     }
+
+    public void sendItemDeliveryConformation(UUID orderPID) {
```

Comment:
You mistyped this. It should be sendItemDeliveryConfirmation. Unless you want to improve the delivery's mental health :D